### ISSUE TITLE
Add Chat with LLM (local/paid with a key) Feature

### ISSUE THREAD

**Issue by mkagenius** [ospfranco/sol#135]
Raycast has it too. Hope some dev can pick it up.

**Comment by ospfranco** [comment]
No, I'm not going to maintain llama through the years, it's a pain in the butt. At most I will accept contribution to run against openAI (or other services that are compatible with their JS library). Anything else will become a burden for me and forgotten by anyone who did the initial implementation.
